src/NServiceBus.TransportTests/When_cancelling_stop.cs

```diff
@@ -57,7 +57,11 @@ public async Task Should_cancel_message_processing(TransportTransactionMode tran
 
             Assert.True(wasCancelled);
             Assert.False(completeContext.OnMessageFailed);
-            Assert.False(completeContext.WasAcknowledged);
+
+            if (transactionMode != TransportTransactionMode.None)
+            {
+                Assert.False(completeContext.WasAcknowledged);
```

Comment:
Should there be no assertion when transactionMode == None? In that case shouldn't we assert the value is true?

(Same comment on other tests too.)